e2e/tests/policy-forms.spec.ts

```
@@ -490,13 +496,169 @@ test.describe('other policy create pages render', () => {
     await expect(page.locator('text=Create TokenRateLimit Policy').first()).toBeVisible({
       timeout: 15_000,
     });
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)